sql/src/test/java/io/crate/analyze/where/WhereClauseAnalyzerTest.java

```diff
@@ -58,109 +51,94 @@
 import org.junit.Before;
 import org.junit.Test;
 
+import java.io.IOException;
 import java.util.ArrayList;
 import java.util.Arrays;
-import java.util.Map;
 
 import static io.crate.testing.SymbolMatchers.isFunction;
 import static io.crate.testing.SymbolMatchers.isLiteral;
 import static io.crate.testing.SymbolMatchers.isReference;
 import static io.crate.testing.TestingHelpers.getFunctions;
 import static io.crate.testing.TestingHelpers.isSQL;
+import static java.util.Collections.singletonList;
 import static org.hamcrest.Matchers.containsInAnyOrder;
 import static org.hamcrest.Matchers.is;
 
 @SuppressWarnings("unchecked")
 public class WhereClauseAnalyzerTest extends CrateDummyClusterServiceUnitTest {
 
-    private static final String GENERATED_COL_TABLE_NAME = "generated_col";
     private static final String DOUBLE_GEN_PARTITIONED_TABLE_NAME = "double_gen_parted";
```

Comment:
similar to `GENERATED_COL_TABLE_NAME`, this is also only used once. think we could remove as well